### PR TITLE
Bug 1528447 - [List][Hero][Card]Truncate long source domains with ellipsis

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 1]
```diff
@@ -56,6 +56,10 @@ $excerpt-line-height: 20;
     .source {
       font-size: 13px;
       color: $teal-80;
+      width: $domain-width;
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: @Mardak trying to find solution for getting rid of hard coded width here, any inputs on how to inherit width from parent container, width: auto doesn’t seem to help. Thanks

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 2]
```diff
@@ -36,6 +36,7 @@ $excerpt-line-height: 20;
     flex-direction: column;
     flex-grow: 1;
     padding: 16px;
+    overflow-wrap: anywhere;
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: Helps to wrap long words in a story that can break alignment which will have high probability once we start supporting localization e.g. I'm a Looooooooooooooooooonngg word

**Comment by punamdahiya** [comment]
@Mardak Submitted PR tht truncates domain links for discovery stream components. Let me know if it will help to run this change by Wolasi for a design review. Thanks!

**Comment by Mardak** [comment]
As noted in https://bugzilla.mozilla.org/show_bug.cgi?id=1528447#c2 looks like -webkit-line-clamp will avoid needing to specify a max-width. Looks like layout change is under review with some revisions, but it seems to be much closer than before.

screen shot 2019-02-25 at 3 06 16 pm

```css
display: -webkit-box;
-webkit-line-clamp: 4 /* or as appropriate */;

/* for source/domain */
word-wrap: anywhere;
overflow: hidden;
```

**Comment by punamdahiya** [comment]
> As noted in https://bugzilla.mozilla.org/show_bug.cgi?id=1528447#c2 looks like -webkit-line-clamp will avoid needing to specify a max-width. Looks like layout change is under review with some revisions, but it seems to be much closer than before.
> 
Agree if https://bugzilla.mozilla.org/show_bug.cgi?id=866102 makes 67 its a much simpler fix, closing this PR in favor of https://bugzilla.mozilla.org/show_bug.cgi?id=1528447#c3

**Pull request closed without merging**